src/rust/ide/src/controller/ide/desktop.rs

```diff
@@ -0,0 +1,109 @@
+//! The Desktop IDE Controller
+//!
+//! See [`crate::controller::ide`] for more detailed description of IDE Controller API.
+
+use crate::prelude::*;
+
+use crate::controller::ide::API;
+use crate::controller::ide::ManagingProjectAPI;
+use crate::controller::ide::StatusNotificationPublisher;
+use crate::controller::ide::Notification;
+use crate::controller::project::ENGINE_VERSION_FOR_NEW_PROJECTS;
+use crate::ide::initializer;
+use crate::notification;
+
+use enso_protocol::project_manager;
+use enso_protocol::project_manager::MissingComponentAction;
+use enso_protocol::project_manager::ProjectName;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+const UNNAMED_PROJECT_NAME:&str = "Unnamed";
+
+
+
+// =============================
+// === The Controller Handle ===
+// =============================
+
+/// The Desktop IDE Controller handle.
+///
+/// The desktop controller has access to the Project Manager, and thus is able to perform all
+/// project management operations using it.
```

Comment:
```suggestion
/// project management operations.
```

Unless there are project management operations that are not using PM?